Command "lint;" not found. Did you mean "lint"?

I am seeing the following error when executing `git push`

" Command "lint;" not found. Did you mean "lint"?"

Running Win10

Honza

Command Line Output
---

$ git push origin master

devtools-core@0.0.1 prepush C:\src\github.com\janodvarko\devtools-core
yarn run lint;

yarn run v0.19.1
error Command "lint;" not found. Did you mean "lint"?
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

npm ERR! Windows_NT 10.0.14393
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "run" "prepush"
npm ERR! node v7.4.0
npm ERR! npm  v4.0.5
npm ERR! code ELIFECYCLE
npm ERR! devtools-core@0.0.1 prepush: `yarn run lint;`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the devtools-core@0.0.1 prepush script 'yarn run lint;'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the devtools-core package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     yarn run lint;
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs devtools-core
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls devtools-core
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     C:\src\github.com\janodvarko\devtools-core\npm-debug.log

husky - pre-push hook failed (add --no-verify to bypass)

error: failed to push some refs to 'git@github.com:janodvarko/devtools-core.git'

--- a/package.json
+++ b/package.json
@@ -6,7 +6,7 @@
     "lint": "yarn run lint-css -s; yarn run lint-js -s",
     "lint-css": "stylelint packages/devtools-launchpad/**/*.css",
     "lint-js": "eslint packages/devtools-launchpad/src packages/devtools-client-adapters/src",
-    "prepush": "yarn run lint;",
+    "prepush": "yarn run lint",
     "postinstall": "lerna bootstrap"
   },
   "repository": {

